docker-compose.yaml

```diff
@@ -0,0 +1,67 @@
+version: "3.2"
+
+services:
+  starknet:
+    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-testnet-sharingan-beta.7.1.experimental.4@sha256:baff5a0aa1075d0f84b83ed366f56928b4a6a0c3316762df14e8b1656f784729
+    ports:
+      - "9615:9615"
+      - "9944:9944"
+      - "30333:30333"
+    command:
+      - "--rpc-external"
+      - "--rpc-methods=unsafe"
+      - "--dev"
+    networks:
+      - internal
+
+  kakarot-deployer:
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
+    environment:
+      - ACCOUNT_ADDRESS=0x3 
+      - PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d 
+      # Custom RPC URL for docker
+      - RPC_URL=http://starknet:9944
+      - EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 
+    volumes:
+      # Since STARKNET_NETWORK is not set, deployments will be saved without network folder
+      - deployments:/app/kakarot/deployments
+    depends_on:
+      - starknet
+    restart: on-failure
+    networks:
+      - internal
+
+  kakarot-rpc:
+    image: ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+    ports:
+      - 3030:3030
+    # Override entrypoint to set custom env variables
+    entrypoint:
+      - "/usr/bin/tini"
+      - "--"
+      - "/bin/sh"
+      - "-c"
+      - |
+        export KAKAROT_ADDRESS=$(jq -r '.kakarot.address' ./deployments/deployments.json);
+        export PROXY_ACCOUNT_CLASS_HASH=$(jq -r '.proxy' ./deployments/declarations.json);
+        /usr/local/bin/kakarot-rpc
```

Comment:
We override the entrypoint but still use `tini` to override it as per [docs](https://github.com/krallin/tini#existing-entrypoint)